gallery/plot_datapoints.py

```diff
@@ -0,0 +1,122 @@
+"""
+==============
+Datapoints FAQ
+==============
+
+The :mod:`torchvision.datapoints` namespace was introduced to ``torchvision.transforms.v2``. This example showcases what
+these datapoints are and how they behave.
+"""
+
+import PIL.Image
+
+import torch
+import torchvision
+
+# We are using BETA APIs, so we deactivate the associated warning, thereby acknowledging that
+# some APIs may slightly change in the future
+torchvision.disable_beta_transforms_warning()
+
+from torchvision import datapoints
+
+
+########################################################################################################################
+# What are datapoints?
+# --------------------
+#
+# Datapoints are zero-copy tensor subclasses:
+
+tensor = torch.rand(3, 256, 256)
+image = datapoints.Image(tensor)
+
+assert isinstance(image, torch.Tensor)
+assert image.data_ptr() == tensor.data_ptr()
+
+
+########################################################################################################################
+# They are needed in :mod:`torchvision.transforms.v2` for the automatic dispatch to the correct kernel.
```

Comment:
This may seem unclear to newbies. 
```suggestion
# They are needed in :mod:`torchvision.transforms.v2` to correctly select appropriate functional method to transform the input data. For example, ...
```

Reply:
No strong opinion but I wonder whether the original sentence was needed at-all. It seems like this is a fairly low-level implementation detail. Maybe we could make it clearer that users this is handled transparently (emphasis mine, not needed in the text)

> **Under the hood**, the v2 transforms rely on datapoints to **transparently** select the appropriate functional methods to transform the input data

Reply:
I have no idea what "transparently" means in that sentence.

Reply:
it means users don't need to worry about it

Reply:
Maybe there is a language barrier here, but I don't feel this words conveys that meaning at all. The "Under the hood" part is fine and if we drop "transparently" the sentiment is still conveyed.

Reply:
I just want to avoid users thinking that these are things they need to worry about. Feel free to use whatever you prefer that doesn't trick users into thinking those details are important for  them to use the v2 transforms

Reply:
Not trying to push for it or prove I'm right, but the confusion made me google it to check my understanding:

![image](https://user-images.githubusercontent.com/1190450/221181943-195143d9-a89e-4407-b7ca-c97772eb42c4.png)

Reply:
I honestly never heard it being used like that. I feel like with the preface stating that most users don't have to touch datapoints at all and prefixing this sentence with "Under the hood", it should be sufficient.